internal/translation/teams/conversion.go

```diff
@@ -0,0 +1,131 @@
+package teams
+
+import (
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/pointer"
+
+	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
+)
+
+type Team struct {
+	Roles    []string
+	TeamName string
+	TeamID   string
+}
+
+type AssignedTeam struct {
+	Usernames []string
+	TeamName  string
+	TeamID    string
+}
+
+type TeamUser struct {
+	Username string
+	UserID   string
+}
+
+func NewTeam(projTeamSpec *akov2.Team, teamID string) *Team {
+	if projTeamSpec == nil {
+		return nil
+	}
+
+	roles := make([]string, 0)
+	for _, role := range projTeamSpec.Roles {
+		roles = append(roles, string(role))
+	}
+
+	team := &Team{
+		Roles:  roles,
+		TeamID: teamID,
```

Comment:
Why is `TeamName` not set here?

Reply:
The only context in which this function is used is for calling to API request AddAllProjectTeams which only requires teamID and Roles.